lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -1273,6 +1274,56 @@ def test_basic(self, mock_emit):
             }
         )
 
+    @ddt.data(
+        *itertools.product(
+            [
+                FORUM_ROLE_ADMINISTRATOR,
+                FORUM_ROLE_MODERATOR,
+                FORUM_ROLE_COMMUNITY_TA,
+                FORUM_ROLE_STUDENT,
+            ],
+            [True, False],
+            [True, False],
+            ["no_group", "group_none", "group_set"],
```

Comment:
minor: i think "group_not_set" might telegraph intent a little more clearly than "no_group", which I am inclined to confuse with "group_none"